index.html

```diff
@@ -3458,70 +3458,79 @@ <h2>Method Operations</h2>
 
     <section>
       <h2>Security Requirements</h2>
+
       <p>
-<a>DID method</a> specifications MUST include their own Security
-Considerations sections. This section MUST consider all the requirements
-mentioned in section 5 of [[RFC3552]] (page 27) for the <a>DID</a> operations
-defined in the specification.
-      </p>
-      <p>
-At least the following forms of attack MUST be considered: eavesdropping,
-replay, message insertion, deletion, modification, and man-in-the-middle.
-Potential denial of service attacks MUST be identified as well.
-      </p>
-      <p>
-This section MUST discuss, per Section 5 of [[RFC3552]], residual risks (such
-as the risks from compromise in a related protocol, incorrect implementation,
-or cipher) after threat mitigation was deployed.
-      </p>
-      <p>
-This section MUST provide integrity protection and update authentication for
-all operations required by Section <a href="#method-operations"></a>.
-      </p>
-      <p>
-If the technology involves authentication, particularly user-host
-authentication, the security of the authentication method MUST be clearly
-specified.
-      </p>
-      <p>
-<a>DID methods</a> MUST discuss the policy mechanism by which <a>DIDs</a> are proven to be uniquely assigned.
-      </p>
+The requirements for all <a>DID method</a> specifications when authoring the
+<em>Security Considerations</em> section are:
       <p>
+
+      <ol>
+        <li>
+A <a>DID method</a> specifications MUST follow all guidelines and normative
+language provided in <a data-cite="RFC3552#section-5">RFC3552: Writing Security
+Considerations Sections</a> for the <a>DID</a> operations defined in the <a>DID
+method</a> specification.
+        </li>
+        <li>
+At least the following forms of attack MUST be documented for the <a>DID</a>
+operations defined in the <a>DID method</a> specification: eavesdropping,
+replay, message insertion, deletion, modification, denial of service,  storage
+or network amplification, and man-in-the-middle.
```

Comment:
```suggestion
The Security Considerations section MUST document the following forms of attack
for the <a>DID</a> operations defined in the <a>DID method</a> specification:
eavesdropping, replay, message insertion, deletion, modification, denial of
service, storage or network amplification, and man-in-the-middle. Other known
forms of attack SHOULD also be documented.
```

Reply:
Yeah, I added a normative statement... I think it is in the spirit of the section, and I can't think of any other way to say it that doesn't drastically weaken what *should* be a requirement.